drwmutex.go

```diff
@@ -438,5 +433,15 @@ func (dm *DRWMutex) DRLocker() sync.Locker {
 
 type drlocker DRWMutex
 
-func (dr *drlocker) Lock()   { (*DRWMutex)(dr).RLock() }
+var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+func randStringRunes(n int) string {
```

Comment:
This is just a rangString not randStringRunes since we don't return []rune

Reply:
Also, I don't know why we are relying on characters, I expect these to be numbers - they are better represented? more performant?

Reply:
They are not human readable anyways as they are random, and I also see this usage only once so not sure what it means..

Reply:
`drLocker` is of type `sync.Locker`, that doesn't take any input for `Lock`. That is the motive for generating a random string. It can be a uuid string also.

Reply:
Changed the function name to `randString` and the string length to be `16`